static/app/views/performance/transactionSummary/header.tsx

```diff
@@ -170,7 +170,7 @@ class TransactionHeader extends Component<Props> {
 
     const tab = (
       <ListLink
-        data-test-id="web-vitals-tab"
+        aria-label={t('Open web vitals tab')}
```

Comment:
Feels like a role would be more suitable here no?

Reply:
for `getRole` be able to work here, the link has to have the attribute `href` and I did not want to add it there, afraid that I would break something, but I just checked and it seems to be fine to have it... code updated 😉